proxy/logging/LogFilter.h

```diff
@@ -382,6 +382,61 @@ LogFilterString::_checkCondition(OperatorFunction f, const char *field_value, si
   return retVal;
 }
 
+static const char *
+findPatternFromParamName(const char *lookup_query_param, const char *pattern)
+{
+  const char *p1 = strstr(lookup_query_param, pattern);
+  while (p1) {
+    // wipe pattern in param name, need search again if find pattern in param value
+    const char *p10 = strchr(p1, '=');
+    if (!p10) {
+      // no "=" after p1, means p1 is not in the param name, and no more param after it
+      p1 = nullptr;
+      break;
+    }
+    const char *p11 = strchr(p1, '&');
+    if (p11 && p10 > p11) {
+      //"=" is after "&" followd by p1, means p1 is not in the param name
+      p1 = strstr(p11, pattern);
+      continue;
+    }
+    // ensure p1 is in the param name now
+    break;
+  }
+  return p1;
+}
+
+static void
+updatePatternForFieldValue(char **field, const char *p1, int field_pos, char *buf_dest)
+{
+  char tmp_text[strlen(buf_dest) + 10];
```

Comment:
Why 10?

Reply:
I think it didn't make sense for "10", I change it to 1.